genesyscloud/resource_genesyscloud_journey_action_map.go

```diff
@@ -636,6 +668,27 @@ func buildSdkOutcomeProbabilityCondition(outcomeProbabilityCondition map[string]
 	}
 }
 
+func flattenOutcomeQuantileCondition(outcomeQuantileCondition *platformclientv2.Outcomequantilecondition) map[string]interface{} {
+	outcomeQuantileConditionMap := make(map[string]interface{})
+	outcomeQuantileConditionMap["outcome_id"] = *outcomeQuantileCondition.OutcomeId
+	outcomeQuantileConditionMap["max_quantile_threshold"] = *typeconv.Float32to64(outcomeQuantileCondition.MaxQuantileThreshold)
+	stringmap.SetValueIfNotNil(outcomeQuantileConditionMap, "fallback_quantile_threshold", typeconv.Float32to64(outcomeQuantileCondition.FallbackQuantileThreshold))
+	return outcomeQuantileConditionMap
+}
+
+func buildSdkOutcomeQuantileCondition(outcomeQuantileCondition map[string]interface{}) *platformclientv2.Outcomequantilecondition { // ???
```

Comment:
Very minor, should we need `// ???` here?

Reply:
Oh, thanks! Missed this!